tests/exec.spec.ts

```diff
@@ -8,8 +8,7 @@
 * SPDX-License-Identifier: EPL-2.0
 **********************************************************************/
 
-import * as fs from "fs";
-import { Command } from "../src/command";
+import {Command} from "../src/command";
```

Comment:
@amiramw What do you use to edit?

Reply:
intellij. 
But i would expect `yarn format-code` to fix this and it doesn't...